dialogflow-cx/README.md

```diff
@@ -31,39 +47,43 @@ to create a webhook for a
    dialogflow.googleapis.com cloudfunctions.googleapis.com`
 1. Setup application default authentication, run: `gcloud auth
    application-default login`
-1. Clone this repo and enter the folder containing the sample code:
-   ```
+1. Clone this repository and enter the folder containing the sample code:
+   ```bash
    git clone https://github.com/GoogleCloudPlatform/contact-center-ai-samples.git
-   cd contact-center-ai-samples/dialogflow-cx-webhook-python
+   cd contact-center-ai-samples/dialogflow-cx
    ```
 1. Deploy the Cloud Function:
    Here `<REGION>` is the Google Cloud region of your function and
    `<PROJECT_ID>` is the Google Cloud project ID that contains your Cloud
-   Function. 
-   ```
+   Function.
+   ```bash
    gcloud functions deploy dialogflow_webhook \
      --runtime python39 \
```

Comment:
The instructions suggest to use the latest Python, 3.10. Should we instruct them earlier to use the latest supported Cloud Functions Python runtime, and link to the docs where that can be found? Otherwise, it's possible there could be an incompatibility when the function gets deployed?

Reply:
That makes sense to me. I ran into the 3.10 compatibility issue when trying to use `gcloud auth`, so my guess is that 3.9 is the latest supported version. I left a comment about that further up the PR, and Nick is on the case :)